proto/provenance/ledger/v1/events.proto

```diff
@@ -65,6 +70,20 @@ message EventLedgerEntryAdded {
   string correlation_id = 3;
 }
 
+// EventLedgerEntryUpdated is emitted when an existing ledger entry is updated.
+// This event is triggered by the UpdateBalances message handler when a ledger
+// entry is successfully updated.
+message EventLedgerEntryUpdated {
```

Comment:
Should the updated balances be included here? Or do we expect listeners to have to query for the updated balance?